src/main/java/uk/gov/companieshouse/company/links/serialization/ChsDeltaDeserializer.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.companieshouse.company.links.serialization;
+
+import java.util.Arrays;
+import org.apache.avro.io.DatumReader;
+import org.apache.avro.io.Decoder;
+import org.apache.avro.io.DecoderFactory;
+import org.apache.avro.reflect.ReflectDatumReader;
+import org.apache.kafka.common.errors.SerializationException;
+import org.apache.kafka.common.serialization.Deserializer;
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.delta.ChsDelta;
+
+//TODO Do we need separate deserializer one for charges and for insolvancy
+@Component
+public class ChsDeltaDeserializer implements Deserializer<ChsDelta> {
+
+    /**
+     * deserialize.
+     */
+    @Override
+    public ChsDelta deserialize(String topic, byte[] data) {
+        try {
+            // TODO: Zahid: Use AvroDeserializer after adding util method in ch-kafka
```

Comment:
We can remove this TODO for now as we agreed with CH to not use AvroDeserialiser in our new components